vhdbuilder/packer/install-dependencies.sh

```diff
@@ -294,9 +294,16 @@ fi
 
 installSGX=${SGX_DEVICE_PLUGIN_INSTALL:-"False"}
 if [[ ${installSGX} == "True" ]]; then
-    SGX_DEVICE_PLUGIN_VERSIONS="1.0"
```

Comment:
please keep the old device plugin image on the VHD until the ccp side change is fully rolled out

Reply:
Done.